nss-day-cohort-50/rare-api-sea-skimmers
# Ts comments

# Description

added comments view to server as well as comments property to post view
added comments create and retrieve


## Type of change
added functionality

# To test

go to postlist in app and click add comments. this will push you to root directory. if you go back to post details and click show comments you will see your new comment at the bottom
